README.md

```diff
@@ -13,7 +13,6 @@ This workflow map aids *users*, *developers* and *maintainers* of the CiscoNodeU
   * [CONTRIBUTING.md](CONTRIBUTING.md) : Contribution guidelines
   * [README-develop-node-utils-APIs.md](docs/README-develop-node-utils-APIs.md) : Developing new CiscoNodeUtils APIs
   * [README-develop-best-practices.md](docs/README-develop-best-practices.md) : Development best practices
-  * [README-develop-add-XR-support.md](docs/README-develop-add-XR-support.md) : How To Update node_utils APIs to Support IOS XR
```

Comment:
Do you want to merge this and and then worry about merging the two main README docs?